challenges/classes.js

```diff
@@ -1,7 +1,24 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+class CuboidMaker {
+    constructor(dimensions={}) {
+      this.length = dimensions.length;
+      this.width = dimensions.width;
+      this.height = dimensions.height;
+    }
+    
+    Volume(){
+        return this.length * this.width * this.height;
+    }
 
+    surfacArea(){
+        return 2 * (this.length * this.width + this.length * this.height + this.width * this.height)
+    }
+  }
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+ console.log(cuboid.volume()); // 100
+ console.log(cuboid.surfaceArea()); // 130
```

Comment:
You seem to be missing a declaration of `cuboid` here. Try using this object for your dimensions for your cuboid variable next time:
```javascript
{ length: 5, width: 4, height: 5 }
```